pkg/cri/store/image/image.go

```diff
@@ -91,6 +92,20 @@ func (s *Store) Update(ctx context.Context, ref string) error {
 // update updates the internal cache. img == nil means that
 // the image does not exist in containerd.
 func (s *Store) update(ref string, img *Image) error {
+	refName := strings.Split(ref, ":")[0]
+	switch refName {
```

Comment:
Will ref be something like `alpine:3.9@sha256:414e0518bb9228d35e4cd5165567fb91d26c6a214e9c95899e1e056fcd349011`?

I searched the code a bit and looks like ref here will only be tag or digest or image ID, so we should be fine.

Reply:
This can be tag or digest or image ID here if I understand correctly.